### ISSUE TITLE
Unable to use init-date attribute

### ISSUE THREAD

**Issue by Mitsukurinadae** [angular-ui/bootstrap#2331]
Hey all,

I'm unable to get the init-date working as an attribute set for the datepicker, does anyone have any examples I can pull to see where it's working? I was looking at the demo but for whatever reason this commit removed the only reference to the initDate in the Javascript:
https://github.com/angular-ui/bootstrap/pull/2184

I copied the plunker on the demo page and tried to edit it:
http://plnkr.co/edit/ZGz7rmv09p6SFVVGB9VX?p=preview

I tried sticking it in the datepicker options as initDate: $scope.initDate
and I also tried to just define init-date: initDate on the attributes of the datepicker.

I've browsed through as many issues as I could and Google didn't have an example as the init-date is pretty new.

Thanks for your help!

**Comment by felixzapata** [comment]
hi @kbaltrinic, for me your solution works in my project, so I'm afraid that until your pull request will be accepted, I'm gonna to put direct your changes into my file and load the datepicker from local instead of bower.

Sorry :(

Thanks.

**Comment by trickpattyFH20** [comment]
@felixzapata after you compiled the changes from pull request #2972, how did you implement "init-date"? is it in the dateOptions object like: $scope.dateOptions = { initDate: ''01/14/1997'' }; or was it on the input element with init-date=" '01/14/1997' " or was there some other method/date format that you used?

**Comment by felixzapata** [comment]
for my case, this way:

``` 
$scope.dateOptions = {
            minDate: new Date(2014, 11, 1),
            maxDate: new Date(2015, 1, 28),
            initDate: new Date(2015, 0, 9)
        };

<input name="date" datepicker-options="options" date-disabled="disabled(date, mode)" min-date="options.minDate" max-date="options.maxDate" datepicker-popup="dd/MM/yyyy" placeholder="dd/mm/aaaa" is-open="opened" ng-model="date" readonly="readonly" type="text">

```

**Comment by felixzapata** [comment]
not only this pull request @trickpattyFH20 would be great ;) , I would need another pull request that it was sent a few months ago [about the navigation](https://github.com/angular-ui/bootstrap/pull/2550).

**Comment by michaellopez** [comment]
+1

**Comment by dciccale** [comment]
Please accept #2438 made like a hundred years ago!

**Comment by rvanbaalen** [comment]
Please be just a little more patient. We're actively going through all PRs at the moment.

**Comment by wesleycho** [comment]
This is truly fixed by 98e2bdfc8b4dd108acdf3fac75a795675889bbc7.
